build/storage/tests/it/requirements.txt

```diff
@@ -0,0 +1,2 @@
+ptf
+scapy
```

Comment:
In Rafal patch we already have `paramiko` and `tenacity`. Will it make sense to add them here as well?

Reply:
added